### PR TITLE
Updated the proto2jsonschema tool to include the examples field

### PR DESCRIPTION
**Pull request by michaelawyu**
This PR patches the post-gen script of the `proto2jsonschema` tool to include an `examples` field in the generated schema, which is one of the drafted generic keywords in JSON schema that provides an array of examples for validation purposes. This field will include an array of paths to the test event data in this repository associated with the schema. We can use this field for automated tests.

### CODE REVIEW

**Review by grant — APPROVED** [review]

File: tools/proto2jsonschema/postgen.js [hunk 1]
```diff
@@ -132,5 +136,39 @@ function getCloudEventType(filepath) {
...
+      } else {
+        return null;
+      }
+    })
+    .filter((value) => {
+      if (value==null) {
+        return false;
+      }
+      return true;
+    })
```
Comment (grant) [inline tools/proto2jsonschema/postgen.js:170]:
```ts
.filter((value) => !value);
```

File: tools/proto2jsonschema/postgen.js [hunk 2]
```diff
@@ -132,5 +136,39 @@ function getCloudEventType(filepath) {
...
+    return [];
+  }
+
+  return filesAndDirs
+    .map((value) => {
+      if (value.isFile() && value.name.endsWith('.json')) {
+        return `https://googleapis.github.io/google-cloudevents/testdata/${removeSuffix}/${value.name}`;
+      } else {
+        return null;
+      }
```
Comment (grant) [inline tools/proto2jsonschema/postgen.js:163]:
```ts
const isJSONFile = value.isFile() && value.name.endsWith('.json');
return isJSONFile ? `https://googleapis.github.io/google-cloudevents/testdata/${removeSuffix}/${value.name}` : null;
```

**Review by grant — APPROVED** [review]

File: tools/proto2jsonschema/postgen.js (hunk 1 above)
Comment (michaelawyu) [reply tools/proto2jsonschema/postgen.js:167]: Done.

File: tools/proto2jsonschema/postgen.js (hunk 2 above)
Comment (michaelawyu) [reply tools/proto2jsonschema/postgen.js]: Done.

**Pull request merged**